### ISSUE TITLE
possibly unsafe calls: assignInNamespace on branch modsForLargeArchives

### ISSUE THREAD

**Issue by achubaty** [PredictiveEcology/reproducible#382]
```r
─  checking R code for possible problems ... [27s/15s] NOTE (14.7s)
   Found the following possibly unsafe calls:
   File ‘reproducible/R/messages.R’:
     assignInNamespace(ns = ns, ".messagePreProcessIndent", paste0(.messagePreProcessIndent, 
       val))
     assignInNamespace(ns = ns, ".messagePreProcessIndent", gsub(paste0(val, 
       "$"), "", .messagePreProcessIndent))
     assignInNamespace(ns = "reproducible", ".messagePreProcessIndent", 
       gsub(paste0(val, "$"), "", .messagePreProcessIndent))
```

Per the documentaiton:

> They should not be used in production code (except perhaps assignInMyNamespace, but see the ‘Note’).

**Comment by achubaty** [comment]
this should be resolved 19a20b760cd2860f11ea039a20a60db3ded18b34, which add a new internal package environment to store all internal messages and message-generating functions.

**Comment by eliotmcintire** [comment]
Thanks @achubaty. This looks great.
